tests/e2e/nodes_scaleup_scaledown.go

```diff
@@ -0,0 +1,284 @@
+		client       clientset.Interface
+		scParameters map[string]string
+	)
+	ginkgo.BeforeEach(func() {
+		namespace = getNamespaceToRunTests(f)
+		client = f.ClientSet
+		bootstrap()
+		scParameters = make(map[string]string)
+	})
+
+	ginkgo.AfterEach(func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+		ginkgo.By(fmt.Sprintf("Deleting all statefulsets in namespace: %v", namespace))
+		fss.DeleteAllStatefulSets(client, namespace)
+		ginkgo.By(fmt.Sprintf("Deleting service nginx in namespace: %v", namespace))
+		err := client.CoreV1().Services(namespace).Delete(ctx, servicename, *metav1.NewDeleteOptions(0))
+		if !apierrors.IsNotFound(err) {
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		}
+	})
+
+	/*
+		Test performs following operations
+
+		Steps
+		1. Create a storage class.
+		2. Create nginx service.
+		3. Create nginx statefulsets with 3 replicas.
+		4. Wait until all Pods are ready and PVCs are bounded with PV.
+		5. Cordon and drain the node
+		6. Wait for the STS pods to reschedule on the available nodes
+		7. Scale down statefulsets to 2 replicas.
+		8. Uncordon the node
+		9. Scale up statefulsets to 3 replicas.
+		7. Scale down statefulsets to 0 replicas and delete all pods.
+		8. Delete all PVCs from the tests namespace.
+		9. Delete the storage class.
+	*/
+	ginkgo.It("Statefulset with Nodes Scale-up and Scale-down", func() {
+		curtime := time.Now().Unix()
+		randomValue := rand.Int()
+		val := strconv.FormatInt(int64(randomValue), 10)
+		val = string(val[1:3])
+		curtimestring := strconv.FormatInt(curtime, 10)
+		scName := "nginx-sc-default-" + curtimestring + val
+
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		dh := drain.Helper{
+			Ctx:                 ctx,
+			Client:              client,
+			Force:               true,
+			IgnoreAllDaemonSets: true,
+			Out:                 ginkgo.GinkgoWriter,
+			ErrOut:              ginkgo.GinkgoWriter,
+		}
+
+		framework.Logf("Fetch the Node Details")
+		nodes, err := client.CoreV1().Nodes().List(ctx, metav1.ListOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Creating StorageClass for Statefulset")
+		if rwxAccessMode {
+			scParameters[scParamFsType] = nfs4FSType
+		}
+		scSpec := getVSphereStorageClassSpec(scName, scParameters, nil, "", "", false)
+		sc, err := client.StorageV1().StorageClasses().Create(ctx, scSpec, metav1.CreateOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		defer func() {
+			err := client.StorageV1().StorageClasses().Delete(ctx, sc.Name, *metav1.NewDeleteOptions(0))
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		}()
+
+		ginkgo.By("Creating service")
+		service := CreateService(namespace, client)
+		defer func() {
+			deleteService(namespace, client, service)
+		}()
+
+		statefulset := GetStatefulSetFromManifest(namespace)
+		ginkgo.By("Creating statefulset")
+		// Check if it is file volumes setups
+		if rwxAccessMode {
```

Comment:
is the environment variable `ACCESS_MODE` documented in md file?

Reply:
Added to readme